customer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Customer extends Model
+{
+    use HasFactory;
+    protected $fillable = [
+        'name',
+        'email',
+        'telephone',
+        'created_at',
+    ];
+
+    public function order()
+    {
+        return $this->hasOne(Order::class);
```

Comment:
One customer can order one or more imobils?
If one, leave as it is.
If no, transform in one to many

Reply:
you are right, one customer can order many imobils
        public function orders() {
        return $this->hasMany(Order::class);
    }